content/Contributing/ForkAndPR/index.md

```diff
@@ -7,9 +7,54 @@ weight = 100
 
 ## Abstract
 
-(coming soon)
+AL Guidelines is a community project, and as such YOU are encouraged to submit corrections and new ideas. In order to get your content included, you must submit a pull request to the GitHub Repository (Located here: https://github.com/microsoft/alguidelines). All Pull Requests are subject to approval by a minimum three admins.
+
+If You are toying with an idea, but You aren't ready to create a document just yet, you are encouraged to create a project discussion thread here: https://github.com/microsoft/alguidelines/discussions
+
+{{% notice warning %}}
+If You haven't worked in collaboration with "external" GitHub repositories before, please familiarize yourself with that process by visiting:
+
+https://docs.github.com/en/pull-requests/collaborating-with-pull-requests
+
+{{% /notice %}}
 
 ## Steps
+Now that You have decided that You are ready to contribute, these are the steps to take.
+{{% notice info %}}
+You can read more about this process here:
+
+https://docs.github.com/en/get-started/quickstart/contributing-to-projects
+{{% /notice %}}
+
 ### Step 1: Fork
+In order to work on the repository, You must [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the repository.
+
+By forking the repository, You essentially create a copy into Your own account. 
+
+Start by going to the GitHub Repository (), and press the Fork ![Fork Button](./fork_button.jpg "Fork")
+
+Once You have successfully forked the repository, go to your own GitHub repository : ![Forked Repository Representation](./ForkedRepro.png "Forked repository")
+
+You are now able to clone your own repository to your local pc and start editing using your favorite editor. [Visual Studio Code](https://code.visualstudio.com/) is perfectly fine for this task.

```

Comment:
I know VS Code is ubiquitous for any developer, But maybe there are people out there who like to proof-read and don't have VS Code available or would need to set it up on the machine they are using (I know, might be not so likely, but anyway).
Github supports basicall VS Code in the browser, by going to the repo and pressing "." - which will open VS Code directly in the tab. So there they could do editing directly in the browser. MAybe it's worth adding a (small) paragraph for that?

Reply:
That will be added later. This is the initial version of this page - I am sure a lot more information will be added